src/net_processing.cpp

```diff
@@ -4822,24 +4822,24 @@ bool PeerManagerImpl::SendMessages(CNode* pto)
                 if (pto->m_tx_relay->lastSentFeeFilter == MAX_FILTER) {
                     // Send the current filter if we sent MAX_FILTER previously
                     // and made it out of IBD.
-                    pto->m_tx_relay->nextSendTimeFeeFilter = count_microseconds(current_time) - 1;
+                    pto->m_tx_relay->m_next_send_feefilter = current_time - std::chrono::microseconds{1};
```

Comment:
I think you can simply assign this to `0`, as the goal is to send immediately

Reply:
Done